### PR TITLE
chore: update action drawer with designer feedback

### PR DESCRIPTION
**Pull request by J-Son89**
Addressed P.r Reviews for feedback on Action Drawer component:
https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?node-id=1531%3A152207&t=YNDSq9VcFXaNlIaC-0

Some things to note on feedback ->

bottom sheet points are to do with the bottom sheet and not this component, as this component is just the content used within it.

Colors are correct

Divider line was not showing in quo2 example because of a configuration, it's set now and working as expected 👍 

Fixed spacing between elements 👍 

Fixed on press state so that it has the correct background highlight color showing 👍

### CODE REVIEW

File: src/quo2/components/drawers/action_drawers.cljs [hunk 1]
```diff
@@ -50,13 +57,15 @@
...
+                     (colors/theme-colors
+                      colors/danger-50 colors/danger-60))}}
          label]
         (when sub-label
           [text/text
            {:size  :paragraph-2
             :style {:color
-                    (colors/theme-colors colors/neutral-50 colors/neutral-40)}}
+                    (colors/theme-colors
+                     colors/neutral-50 colors/neutral-40)}}
```
Comment (OmarBasem) [inline src/quo2/components/drawers/action_drawers.cljs]: Why breakline here?

File: src/quo2/components/drawers/action_drawers.cljs [hunk 2]
```diff
@@ -50,13 +57,15 @@
           :weight :medium
           :style  {:color
                    (when danger?
-                     (colors/theme-colors colors/danger-50 colors/danger-60))}}
+                     (colors/theme-colors
+                      colors/danger-50 colors/danger-60))}}
```
Comment (OmarBasem) [inline src/quo2/components/drawers/action_drawers.cljs]: and here?

File: src/quo2/components/drawers/action_drawers.cljs [hunk 3]
```diff
@@ -27,10 +28,16 @@
...
+                                               :margin-horizontal 8}
+                              :underlay-color (colors/theme-colors
+                                               colors/neutral-5 colors/neutral-90)
+                              :on-press       on-press}
       [rn/view {:style
-                {:height            (if sub-label 56 47)
-                 :margin-horizontal 20
+                {:height (if sub-label 58 50)
+
+                 :margin-horizontal 12
```
Comment (OmarBasem) [inline src/quo2/components/drawers/action_drawers.cljs:39]: Empty space?

File: src/quo2/components/drawers/action_drawers.cljs [hunk 4]
```diff
@@ -27,10 +28,16 @@
   (when action-props
     [:<> {:key label}
      (when add-divider? [divider])
-     [rn/touchable-opacity {:on-press on-press}
+     [rn/touchable-highlight {:style          {:border-radius     12
+                                               :height (if sub-label 58 50)
+                                               :margin-horizontal 8}
+                              :underlay-color (colors/theme-colors
+                                               colors/neutral-5 colors/neutral-90)
```
Comment (OmarBasem) [inline src/quo2/components/drawers/action_drawers.cljs]: Why breakline here?

**Review by OmarBasem — APPROVED** [review]
Non blocking comments

File: src/quo2/components/drawers/action_drawers.cljs (hunk 1 above)
Comment (J-Son89) [reply src/quo2/components/drawers/action_drawers.cljs]: seemed like a good idea, I can revert :)

File: src/quo2/components/drawers/action_drawers.cljs (hunk 2 above)
Comment (J-Son89) [reply src/quo2/components/drawers/action_drawers.cljs]: same ^^

File: src/quo2/components/drawers/action_drawers.cljs (hunk 3 above)
Comment (J-Son89) [reply src/quo2/components/drawers/action_drawers.cljs:39]: will remove 👍

**Review by flexsurfer — APPROVED** [review]

**Comment by J-Son89** [comment]
@VladimrLitvinenko this is just related to a component in quo2, so I don't think it needs QA? 🤔

**Comment by VolodLytvynenko** [comment]
> @VladimrLitvinenko this is just related to a component in quo2, so I don't think it needs QA? 🤔

ok.  Thank you

**Pull request merged**